.babelrc

```diff
@@ -1,21 +1,26 @@
 {
   "presets": [
-    ["@babel/preset-env", {
-      "modules": false
-    }]
+    [
+      "@babel/preset-env",
+      {
+        "modules": false,
+        "bugfixes": true
```

Comment:
See https://babeljs.io/blog/2020/03/16/7.9.0

It didn't end up saving any bytes, unfortunately :'(. But still good to have

Reply:
Correction - I had this configured wrong. See fix in https://github.com/single-spa/single-spa/pull/495/commits/b5baa581c78dc57e6885538d04fa8d3cbb08b235. This saved about 300 bytes